agent/util-scripts/register-tool-set

```diff
@@ -94,9 +95,20 @@ default_tool_set=$(getconf.py -l default-tool-set pbench/tools)
 case "$toolset" in
 	default)
 	for i in $default_tool_set; do
-		register-tool --name=$i $remote $label $group -- --interval="$interval"
+		if [ ! -z $remote ] ; then
+			# ensure pbench packages are installed on remote system 
+			ssh $ssh_opts $(echo $remote | cut -d"=" -f2) yum -y install "$pbench_agent"
```

Comment:
I would really prefer we only install pbench-agent outside of the pbench scripts.  I don't think the burden to install before testing is too high.  I also do not want to use yum or other package commands directly in the benchmark or tools scripts.